src/ucp/core/ucp_am.c

```diff
@@ -1166,10 +1167,10 @@ ucp_am_handler_common(ucp_worker_h worker, ucp_am_hdr_t *am_hdr, size_t hdr_size
          * right before the data, where user header resides).
          */
         status = ucp_recv_desc_init(worker, data,
-                                    total_length - hdr_size - am_hdr->header_length,
-                                    0,
-                                    0, /* pass as a const */
-                                    0, 0, -hdr_size, &desc);
+                                    total_length - hdr_size -
+                                        am_hdr->header_length,
```

Comment:
do we really need to use such alignment?

Reply:
clang format?

Reply:
yeah, I mentioned this in #6126. Originally clang suggested this formatting, but when I fixed it according to the suggestion clang is not happy again for some reason(